migrations/sqls/20200214162622-event-add-bookshelf-default-timestamp-columns-up.sql

```diff
@@ -0,0 +1,4 @@
+/* Replace with your SQL commands */
+ALTER TABLE water.events ADD COLUMN created_at timestamp DEFAULT now();
+ALTER TABLE water.events ADD COLUMN updated_at timestamp DEFAULT null;
+UPDATE water.events SET created_at = created, updated_at = modified;
```

Comment:
only adding columns for the default Bookshelf managed timestamps. I have deliberately not removed the existing columns because as you say that will break things. Once we have removed the deprecated event service we can remove the modified and updated columns

Reply:
OK i just spotted how you can define custom timestamp columns instead of the default ones. e.g. hasTimestamps: ['date_created', 'date_updated'] Thanks

Reply:
done